### DISCUSSION TITLE
Import structure from ".cif" file

### DISCUSSION THREAD

**Discussion by rybakov-ks** [SINGROUP/dscribe#94] [Q&A]
Hello everyone. 
I'm trying to replicate the example from the "Ewald sum matrix" section. Everything worked out, I determined the total electrostatic energy for aluminum using DScribe and pymatgen, the results are in good agreement with each other (-1382.9601187179453 eV and -1382.9601186780128 eV ). However, when I try to transfer the aluminum crystal structure in a "**.cif**" file format, I get different values (-2055.093 eV and -5510.421 eV ) and this happens for all the "**.cif**" and "**.vasp**" files I've used. What am I doing wrong?

```
from ase.build import bulk
from ase.io import read
from dscribe.descriptors import EwaldSumMatrix
import scipy
import math
from pymatgen.core import Structure
from pymatgen.analysis.ewald import EwaldSummation

#Structure import
al = bulk("Al", "fcc", a=4.0339)
#or
#al = read("9008460.cif")

#DScribe
ems = EwaldSumMatrix(
    n_atoms_max=168,
    permutation="none",
    flatten=False
)
ems_out = ems.create(al, accuracy=1e-10)
total_energy = ems_out[0, 0] + ems_out[1, 1] + ems_out[0, 1]
conversion = 1e10 * scipy.constants.e / (4 * math.pi * scipy.constants.epsilon_0)
total_energy *= conversion
print(total_energy)

#pymatgen
structure = Structure(
    lattice=al.get_cell(),
    species=al.get_atomic_numbers(),
    coords=al.get_scaled_positions(),
)
structure.add_oxidation_state_by_site(al.get_atomic_numbers())
ewald = EwaldSummation(structure)
energy = ewald.total_energy
print(energy)
```

[9008460.zip](https://github.com/SINGROUP/dscribe/files/9417891/9008460.zip)

**Comment by FlorianTseng** [comment]
Have you solved this problem?

if you try to:
# Structure import
al = bulk("Al", "fcc", a=4.0339)
print(al)
# or
al = read("9008460.cif")
print(al)

then the result is:
Atoms(symbols='Al', pbc=True, cell=[[0.0, 2.01695, 2.01695], [2.01695, 0.0, 2.01695], [2.01695, 2.01695, 0.0]])
Atoms(symbols='Al4', pbc=True, cell=[4.04958, 4.04958, 4.04958], spacegroup_kinds=...)

It might be sth. wrong with your cif file.
